scripts/runtime/environment

```diff
@@ -15,104 +17,81 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# This script is written into .bashrc to set up toolchains when enter the docker, like:
-# docker run -it \
-#   -e TA_ARCH=aarch64 \
-#   -e CA_ARCH=arm \
-#   -e STD=y \
-#   teaclave-dev bash
+# =============================================================================
+# config/environment (Main configuration - loads active configs)
+# =============================================================================
+
+# Exit on any error
+set -euo pipefail
```

Comment:
The `-e` causes docker container shell exit on any error. e.g. 
```
~/teaclave_sdk_src# switch_config
Error: No option specified
[The docker container stopped]
```
And other errors such as "vim command not found".
Recommend remove the '-e'.

Reply:
These lines will be removed.